### ISSUE TITLE
TypeError: Cannot read properties of null (reading '1')

### ISSUE THREAD

**Issue by Vanilor** [MrPrimate/ddb-importer#232]
Hello!

While trying to import a character I receive this error:

> TypeError: Cannot read properties of null (reading '1')

The logged exception for it:

```
logger.js:67 DDB Importer | ERR > TypeError: Cannot read properties of null (reading '1')
    at URL/modules/ddb-importer/dist/main.js:20592:45
    at Array.forEach (<anonymous>)
    at getSpellScaling (URL/modules/ddb-importer/dist/main.js:20540:8)
    at parseSpell (URL/modules/ddb-importer/dist/main.js:20751:24)
    at URL/modules/ddb-importer/dist/main.js:33946:20
    at Array.forEach (<anonymous>)
    at URL/modules/ddb-importer/dist/main.js:33912:24
    at Array.forEach (<anonymous>)
    at getCharacterSpells (URL/modules/ddb-importer/dist/main.js:33896:29)
    at parseJson (URL/modules/ddb-importer/dist/main.js:34459:18)
    at async getCharacterData (URL/modules/ddb-importer/dist/main.js:36146:25)
    at async HTMLButtonElement.<anonymous> (URL/modules/ddb-importer/dist/main.js:36477:33)
```

Foundry version: Version 9 - Build 269
DnD5e System version: 1.6.0
D&D Beyond Importer version: 2.9.34

**Comment by MrPrimate** [comment]
Can I get a character URL?

**Comment by Vanilor** [comment]
Hi, sure. This is the [character sheet](https://www.dndbeyond.com/characters/50382768).

**Comment by MrPrimate** [comment]
This should be fixed in v2.9.35 - it looks like they changed some of the data model that I didn't catch, but was only showing itself in homebrew spells.
